webapp/channels/src/components/searchable_channel_list.tsx

```diff
@@ -223,6 +223,21 @@ export class SearchableChannelList extends React.PureComponent<Props, State> {
                 </div>
                 <div className='more-modal__actions'>
                     {joinViewChannelButton}
+                    {this.state.joiningChannel === channel.id && !this.isMemberOfChannel(channel.id) && (
+                        <span
+                            className='sr-only'
+                            role='alert'
+                            aria-live='assertive'
```

Comment:
You can actually remove `aria-live` from this because the `alert` role automatically gets that. 0/5 on if you want to remove that though or make it extra clear